src/patternfly/components/Table/table.scss

```diff
@@ -593,6 +593,14 @@
       border-block-end: 0;
     }
   }
+
+  .#{$action-list}.pf-m-icons {
+    --#{$action-list}--m-icons--ColumnGap: 0;
+  }
```

Comment:
This needs to be better scoped - it compiles to `.pf-v6-c-table .pf-v6-c-action-list.pf-m-icons`, which will apply to any use of `.pf-v6-c-action-list.pf-m-icons` anywhere in a table.

If all this is doing is putting 2 things side by side in a flex layout, could it just be a local style to the table?

Reply:
I added action list and modified the spacer as you suggested. Also left a comment regarding the icon- group spacing being inconsistent with all other icon groups. I disagreement that it's scoped poorly.